_includes/config/split-deploy/split-deploy-overview.md

```diff
@@ -22,4 +22,4 @@ Staging system
 :	_Optional_. You can optionally set up a staging system to use for final testing of all integrated code, including User Acceptance Testing (UAT). Set up a staging system the same way you set up a production system. Except for the fact that staging is not your live store and doesn't process orders from customers, it's identical to production.
 
 Production system
-:	Your live store. You should make minimal configuration changes here and no changes to websites, stores, store views, products, or the catalog. You should make all those types of changes in your development system.
+:	Your live store. You should make minimal configuration changes here and no changes to websites, stores, store views, product view settings, or the catalog. You should make all those types of changes in your development system.
```

Comment:
Why only in product view settings and not products in general. 

We can say the same of category view settings?

Reply:
@osrecio Thank for information. I appreciate for that i have updated to
"Your live store. You should make minimal configuration changes here and no changes to websites, stores, store views, products, product view settings, or the catalog, categories, category view settings. You should make all those types of changes in your development system."

Reply:
@osrecio Changes has done.